pages/osx/hashcat.md

```diff
@@ -0,0 +1,24 @@
+# hashcat
+
+> Password recovery utility
+> More information: <https://hashcat.net/wiki/>.
+
+- Wordlist
+
+`hashcat -a 0 -m 400 example400.hash example.dict`
+
+- Wordlist + Rules
+
+`hashcat -a 0 -m 0 example0.hash example.dict -r rules/best64.rule`
```

Comment:
```suggestion
- Wordlist + Rules:

`hashcat --attack-mode={{0}} --hash-type={{0}} {{example0.hash}} {{example.dict}} --rules-file={{path/to/rules_best64.rule}}`
```

Almost the same issue as above.
Mandatory fix: [broken placeholder](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): generic path expected.